src/key_io.cpp

```diff
@@ -146,6 +146,9 @@ CTxDestination DecodeDestination(const std::string& str, const CChainParams& par
         // The rest of the symbols are converted witness program bytes.
         data.reserve(((dec.data.size() - 1) * 5) / 8);
         if (ConvertBits<5, 8, false>([&](unsigned char c) { data.push_back(c); }, dec.data.begin() + 1, dec.data.end())) {
+
+            const std::string byteStr = data.size() == 1 ? "byte" : "bytes";
```

Comment:
Applied, committed, squashed, and pushed. 

- Is there any need for `const` keyword with std::string_view since it is read only? 

- Snake case has been remedied.